chapter09/mdw/Ex05.pl

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+$^I= ".out";
+
+my $line = 1;
+
+while (<>) {
+	if ( $line == 2 ) {
+		if ( !(/\#\# Copyright \(C\) 2015 by mdw/) ){
```

Comment:
could rewrite as `$line == 2 &&  !(/\#\# Copyright \(C\) 2015 by mdw/)`, but I'm still not sure about hardcode with line 2, what if the copyright is at line 3?